### PR TITLE
Add lincstation n1 to cases

### PR DESCRIPTION
**Pull request by Squidly271**

### CODE REVIEW

File: emhttp/plugins/dynamix/font-cases.woff
Comment (zackspear) [inline emhttp/plugins/dynamix/font-cases.woff:1]: Please move into `emhttp/plugins/dynamix/styles`

File: emhttp/plugins/dynamix/styles/default-cases.css [hunk 1]
```diff
@@ -71,4 +71,5 @@
 .case-vivopc:before{content:'\e925'}
 .case-vm:before{content:'\e920'}
 .case-x3650-M3:before{content:'\e924'}
-.case-zalman-ms800:before{content:'\e947'}
\ No newline at end of file
+.case-zalman-ms800:before{content:'\e947'}
+.case-lincstation-n1:before{content:'\e948';font-weight:400}
```
Comment (zackspear) [inline emhttp/plugins/dynamix/styles/default-cases.css:75]: Is `font-weight: 400` required here? It would be the first case to need this.

**Review by zackspear — CHANGES_REQUESTED** [review]

File: emhttp/plugins/dynamix/styles/default-cases.css (hunk 1 above)
Comment (Squidly271) [reply emhttp/plugins/dynamix/styles/default-cases.css]: Yeah, Been awhile since I completely messed up a PR lol

Comment (zackspear) [reply emhttp/plugins/dynamix/styles/default-cases.css]: Been there plenty of times ;)

**Pull request closed without merging**